src/pages/Home/components/HeroSection/components/modalPlayer/index.jsx

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types'
```

Comment:
good job!  But it would be good to remove all functionality related to modal functionality becouse it will be added at
https://github.com/weblodz/folio/pull/15